io.github.mujx.Nheko.json

```diff
@@ -0,0 +1,210 @@
+{
+  "id": "io.github.mujx.Nheko",
+  "command": "nheko",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.gcc8"
+  ],
+  "writable-sdk": true,
+  "build-options": {
+    "env": {
+      "CC": "/usr/lib/sdk/gcc8/bin/gcc",
+      "CXX": "/usr/lib/sdk/gcc8/bin/g++",
+      "LD_LIBRARY_PATH": "/usr/lib/sdk/gcc8/lib",
+      "V": "1"
+    }
+  },
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=home",
+    "--share=ipc",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "build-commands": [
+        "make install"
+      ],
+      "buildsystem": "simple",
```

Comment:
Don't bypass flatpak-builder build systems, lmdb can be build like this: https://github.com/flathub/org.kde.elisa/blob/0ccfce1ee0f6a1e906e9f3300233f60bb5089e18/org.kde.elisa.json#L66